src/layers/basic.jl

```diff
@@ -253,3 +253,50 @@ end
 function Base.show(io::IO, b::SkipConnection)
   print(io, "SkipConnection(", b.layers, ", ", b.connection, ")")
 end
+
+"""
+    Parallel(connection, layers...)
+
+Create a new 'Parallel' layer that passes a single input array each path in
+`layers`, combining the output of each path with `connection`.
+`connection` should be a reducible operator (i.e. it can be passed to `Base.reduce`).
+
+# Examples
+
+```jldoctest
+julia> model = Chain(
+  Dense(1, 1),
+  Parallel(
+    vcat,
+    Dense(1, 1),
+    Dense(1, 3),
+    Chain(
+      Dense(1, 5),
+      Dense(5, 2),
+    )
+  ),
+  Dense(6, 1)
+);
+julia> model(rand(1))
+Float32[0.27]
+```
+"""
+struct Parallel{F, T}
+  connection::F
+  layers::T
+end
+
+Parallel(connection, layers...) = Parallel(connection, layers)
+
+Flux.@functor Parallel
```

Comment:
```suggestion
@functor Parallel
```

I think this keeps better consistency with the rest of the file.